docs/Web/Browser/02-reflow-repaint.md

```diff
@@ -7,7 +7,7 @@ description: Introduction to the reflow and repaint.
 How the browser renders the document?
 
 1. Receives the data (bytes) from the server.
-1. Parses and converts into tokens (<, TagName, Attribute, AttributeValue, >).
+1. Parses and converts into tokens (\<, TagName, Attribute, AttributeValue, \>).
```

Comment:
you don't need to escape the `>` char normally

Reply:
If I don't escape `<` an error will occur. And the error tells me to go to line 6 column 38 which is an empty line.

![image](https://github.com/xiaohai-huang/learning-notes/assets/56380096/ce48b8ab-997c-4b4a-add5-8ecd30112a48)

Reply:
line number is wrong, it's a Docusaurus bug I'll fix

Yes you need to escape `<`, but not `>`

Playground: 

<img width="998" alt="CleanShot 2023-09-29 at 16 19 17@2x" src="https://github.com/xiaohai-huang/learning-notes/assets/749374/5e0f8f49-2d02-44ae-8126-a7397c372eba">